### PR TITLE
Add tests for mediacapture-image

### PR DESCRIPTION
**Pull request by xiuqijix**
Cover following checkpoints:

- getPhotoCapabilities() should throw "OperationError" when the MediaStreamTrack being ended asynchronously
- getPhotoSettings() should throw "OperationError" when the MediaStreamTrack being ended asynchronously

### CODE REVIEW

File: mediacapture-image/getPhotoCapabilities.html [hunk 1]
```diff
@@ -58,4 +58,19 @@
...
+  context.fillRect(0, 0, 10, 10);
+  let stream = canvas.captureStream();
+  let videoTrack = stream.getVideoTracks()[0];
+
+  let capturer = new ImageCapture(videoTrack);
+  capturer.getPhotoCapabilities()
+  .then(t.step_func_done(() => assert_unreached('should throw "OperationError"')))
+  .catch(t.step_func_done(e => assert_equals(e.name, 'OperationError')))
+  videoTrack.stop();
+}, 'throw "OperationError" when the MediaStreamTrack being ended asynchronously');
```
Comment (reillyeon) [inline mediacapture-image/getPhotoCapabilities.html]: 'throw "OperationError" when the MediaStreamTrack is stopped while getting photo capabilities'

File: mediacapture-image/getPhotoSettings.html [hunk 2]
```diff
@@ -43,4 +43,19 @@
...
+  context.fillRect(0, 0, 10, 10);
+  let stream = canvas.captureStream();
+  let videoTrack = stream.getVideoTracks()[0];
+
+  let capturer = new ImageCapture(videoTrack);
+  capturer.getPhotoSettings()
+  .then(t.step_func_done(() => assert_unreached('should throw "OperationError"')))
+  .catch(t.step_func_done(e => assert_equals(e.name, 'OperationError')))
+  videoTrack.stop();
+}, 'throw "OperationError" when the MediaStreamTrack being ended asynchronously');
```
Comment (reillyeon) [inline mediacapture-image/getPhotoSettings.html]: 'throw "OperationError" when the MediaStreamTrack is stopped while getting photo settings'

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: mediacapture-image/getPhotoCapabilities.html (hunk 1 above)
Comment (kaixinjxq) [reply mediacapture-image/getPhotoCapabilities.html]: @reillyeon, thanks for your review! Updated.

**Review by reillyeon — APPROVED** [review]

**Comment by kaixinjxq** [comment]
@reillyeon, thanks for your review!
These two tests fail in Chrome, I've filed a bug to track it https://bugs.chromium.org/p/chromium/issues/detail?id=884125

**Pull request merged**